test/User-test.js

```diff
@@ -0,0 +1,63 @@
+    sampleIngredientsData;
+    sampleUsersData;
+    sampleRecipeData;
+  });
+
+  it('should have a name property', () => {
+    expect(user.name).to.equal("Saige O'Kon");
+  });
+  it('should have an id property', () => {
+    expect(user.id).to.equal(1)
+  });
+  it('should have a list of ingredients in pantry', () => {
+    expect(user.pantry).to.deep.equal([{"ingredient": 11297,"amount": 4},{"ingredient": 1082047,"amount": 10},{"ingredient": 20081,"amount": 5},{"ingredient": 11215,"amount": 5},{"ingredient": 2047,"amount": 6},{"ingredient": 1123,"amount": 8},{"ingredient": 11282,"amount": 4},{"ingredient": 6172,"amount": 2},{"ingredient": 2044,"amount": 2},{"ingredient": 2050,"amount": 4},{"ingredient": 1032009,"amount": 3},{"ingredient": 5114,"amount": 3},{"ingredient": 1017,"amount": 2},{"ingredient": 18371,"amount": 7},{"ingredient": 1001,"amount": 6},{"ingredient": 99223,"amount": 2},{"ingredient": 1230,"amount": 2},{"ingredient": 9152,"amount": 4},{"ingredient": 10611282,"amount": 2},{"ingredient": 93607,"amount": 2},{"ingredient": 14106,"amount": 4},{"ingredient": 1077,"amount": 4},{"ingredient": 6150,"amount": 2},{"ingredient": 1124,"amount": 2},{"ingredient": 10011693,"amount": 4},{"ingredient": 1102047,"amount": 2},{"ingredient": 19206,"amount": 2},{"ingredient": 1145,"amount": 4},{"ingredient": 1002030,"amount": 4},{"ingredient": 12061,"amount": 2},{"ingredient": 19335,"amount": 4},{"ingredient": 15152,"amount": 3},{"ingredient": 9003,"amount": 2},{"ingredient": 18372,"amount": 3},{"ingredient": 2027,"amount": 2}])
+  });
+  it('should have a list of recipes to cook', () => {
+    expect(user.recipesToCook).to.deep.equal([])
+  });
+  it('should have a method to add recipes to cook', () => {
```

Comment:
Does this add a recipe if it is already there? If not, should we add a conditional in the method? If so, should we test that as our sad path here?